shared/src/commonMain/composeResources/drawable/kotlinconf_by_jetbrains.xml

```diff
@@ -0,0 +1,73 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This drawable needs a separate variant for dark mode. Can't find it in the design at the moment though.

<img width="587" alt="image" src="https://github.com/user-attachments/assets/2ffe93a7-5380-4221-9a96-8d9ba4012fea" />